src/AccessibilityInsights.SharedUx/Controls/EventConfigTabControl.xaml.cs

```diff
@@ -227,6 +227,16 @@ private void TreeViewItem_KeyDown(object sender, KeyEventArgs e)
                 {
                     evm.IsChecked = !evm.IsChecked;
                 }
+
+                if (AutomationPeer.ListenerExists(AutomationEvents.LiveRegionChanged))
+                {
+                    TreeViewItemAutomationPeer peer = UIElementAutomationPeer.FromElement(sender as TreeViewItem) as TreeViewItemAutomationPeer;
+                    if (peer != null)
+                    {
+                        peer.RaiseAutomationEvent(AutomationEvents.LiveRegionChanged);
```

Comment:
Can we do this via the `Toggle` pattern instead?

Reply:
@sfoslund, @RobGallo had a branch where he was experimenting with implementing the `Toggle` pattern here. That would be the preferred option if it's at all possible. 

`RaiseAutomationEvent` is sort of a backup option when we can't find a more canonical form

Reply:
Closing PR this as I'm looking at implementing the toggle pattern instead.